rules/python_rules.build_defs

```diff
@@ -336,7 +336,7 @@ def pip_library(name:str, version:str, hashes:list=None, package_name:str=None,
       version (str): Specific version of the package to install.
       hashes (list): List of acceptable hashes for this target.
       package_name (str): Name of the pip package to install. Defaults to the same as 'name'.
-      outs (list): List of output files / directories. Defaults to [name].
+      outs (list): List of output files / directories. Defaults to [name]. If subdirectory is used outs[0] is used as the prefix.
```

Comment:
Does it make sense to just use the rule name? That way `//third_party/pythong:foo` always outputs to `plz-out/gen/third_party/python/foo/...`